packages/mobile/src/screens/collection-screen/CollectionScreenDetailsTile.tsx

```diff
@@ -89,20 +87,9 @@ export const CollectionScreenDetailsTile = ({
 
   const isReachable = useSelector(getIsReachable)
 
-  const collectionUid = useSelector(getCollectionUid)
-  const userUid = useSelector(getUserUid)
-
   const fetchLineup = useCallback(() => {
-    dispatch(resetCollection(collectionUid, userUid))
-
-    // Need to refetch the collection after resetting
-    // Will pull from cache if it exists
-    // TODO: fix this for smart collections
-    if (typeof collectionId === 'number') {
-      dispatch(fetchCollection(collectionId))
-    }
-    dispatch(tracksActions.fetchLineupMetadatas(0, 200, false, undefined))
-  }, [dispatch, collectionUid, userUid, collectionId])
+    dispatch(resetAndFetchCollection(collectionId))
```

Comment:
this doesn't match the name in the actions file `resetAndFetchCollectionTracks`

Reply:
Updated that right at the end, fixed!